test/shell/cli_test.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bats
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+start_adapter() {
+  run ./etcd-adapter
+  [ "$status" -eq 0 ]
+  sleep "$1"
+}
+
+stop_adapter() {
+  run pkill -9 etcd-adapter
```

Comment:
Do not use such a mandatory way stop the process.

Reply:
Hi, @tokers.

🤔 I'm sorry I'm a bit confused, the idea of this code comes from `apisix-dashboard`'s CLI test (`apisix-dashboard` uses systemd to manage process), in this project we haven't introduced systemd related code yet and it may not be supported in the future, so I'm currently using this way to force end process.

If there is a better way, I hope you can guide me. Thanks

Reply:
We can just handle the `SIGINT` and `SIGTERM` signals in our program, so that we can stop the program gracefully.

Reply:
So, we use `SIGTERM` instead of `SIGKILL` to achieve a graceful exit of the program.

Reply:
changed, Now adapter will receive `SIGINT` and `SIGTERM` signal, print the log and exit. The test script has also been changed to send SIGTERM by default, but since the current logs are not output to a file, we can't check it for now. It will check later if the `See you next time!` are printed out.

ping @tokers